bindings/python3/econf.py

```diff
@@ -0,0 +1,996 @@
+"""
+Econf provides functionality for interacting with Key-Value config files, like getting and setting values for read config files.
+
+For more information please have a look at the API
+"""
+import ctypes.util
+from enum import Enum
+from dataclasses import dataclass
+from typing import *
+from ctypes import *
```

Comment:
Not now. But in a future PR we should remove this and export the specific symbols.